tasks/main.yml

```diff
@@ -14,7 +14,7 @@
 - name: User creation
   user:
     name: "{{item.username}}"
-    group: "{{item.username if users_create_per_user_group else users_group}}"
+    group: "{{item.username if users_create_per_user_group else (item.group if define_individual_user_group else users_group)}}"
```

Comment:
I think `define_individual_user_group` might be unnecessary. What if `item.group` just took precedence when defined?

Something like:

    "{{item.group if 'group' in item else (item.username if users_create_per_user_group else users_group)}}"

This would also require updating the `Per-user group creation` task condition:

    when: users_create_per_user_group and 'group' not in item